specifyweb/specify/migrations/0012_coordinate_fields_fix.py

```diff
@@ -0,0 +1,38 @@
+# Generated by Django 3.2.15 on 2024-11-02 00:46
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('specify', '0011_cascading_tree_nodes'),
```

Comment:
TODO: Update this with the last current migration in the specify app before merge